dynamiq/nodes/agents/base.py

```diff
@@ -147,6 +152,11 @@ def execute(
             self.memory.add(role=MessageRole.USER, content=input_data.get("input"), metadata=metadata)
             self._retrieve_memory(input_data)
 
+        files = input_data.get("files", self.files)
```

Comment:
Currently in our flows and nodes we support BytesIO for file transferring across nodes. Here it seems you working with another structure.
Do you know what our plan is with integration files to Agents?

Examples of Whisper where handles file_name, file_content_type: https://github.com/dynamiq-ai/dynamiq/blob/main/dynamiq/nodes/audio/whisper.py#L67

Reply:
Files also can contain some file description from user, please look at example:
1.  https://github.com/dynamiq-ai/dynamiq/blob/e00600210ae204dd2f376771bd0c86f85f1df553/examples/use_case_files/use_file_tool_example.py#L43
2. https://github.com/dynamiq-ai/dynamiq/blob/e00600210ae204dd2f376771bd0c86f85f1df553/examples/agents_use_cases/agent_analyst_with_files.py#L79

And if you have any suggestions how to do it in proper way, we can change or modify this

For file we can use them from local paths (but if this redundant, I can keep only bytes)

Reply:
I've done such changes, so keep only bytes and bytesIO